.github/workflows/test.yml

```
@@ -69,13 +69,26 @@ jobs:
         name: compiled-packages
     - name: Extract compiled artifacts
       run: tar -xzf compiled.tar.gz
+    # pnpm-registry is the Rust HTTP server the test harness spawns
+    # in place of verdaccio (see __utils__/jest-config/with-registry/globalSetup.js).
+    # Built per-platform here because the binary is native code and the
+    # ubuntu and windows test jobs each need their own.
+    - name: Install Rust toolchain
+      uses: ./.github/actions/rustup
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/rustup": line 7: unexpected key "type" for definition of input "shared-key"

(action)